package.json

```diff
@@ -38,6 +38,8 @@
     "@typescript-eslint/parser": "4.28.4",
     "compressjs": "1.0.3",
     "cross-env": "5.2.0",
+    "esbuild": "0.12.15",
+    "esbuild-jest": "0.5.0",
```

Comment:
do we need both esbuild-jest and ts-jest?

Reply:
no. weirdly the tests only pass with ts-jest and one test fails with esbuild-jest. I'll make a separate PR after this to demonstrate